diff --git a/dist/build/utils.js b/dist/build/utils.js
index 6ec0fe0e0b2e19e2e8151810b4c6788e08dcd8a4..6f9c2afffbb6c09b6896da97b3e5ff7a4cb4b79b 100644
--- a/dist/build/utils.js
+++ b/dist/build/utils.js
@@ -1015,14 +1015,15 @@ async function copyTracedFiles(dir, distDir, pageKeys, appPageKeys, tracingRoot,
                         await _fs.promises.symlink(symlink, fileOutputPath);
                     } catch (err) {
                         // Windows doesn't support creating symlinks without elevated privileges, unless
-                        // "Developer Mode" is turned on. If we failed to create a symlink due to EPERM, try
-                        // creating a junction point instead.
+                        // "Developer Mode" is turned on. If we failed to create a symlink due to EPERM (or
+                        // ENOENT when the untyped symlink targets a directory on some Windows setups),
+                        // try creating a junction point instead.
                         //
                         // Ideally we'd just preserve the input file type (junction point or symlink), but
                         // there's no API in node.js to differentiate between a junction point and a symlink,
                         // so we just try making a symlink first. Symlinks are preferred because they support
                         // relative paths and non-directory (file) targets.
-                        if (process.platform === 'win32' && err.code === 'EPERM' && _path.default.isAbsolute(symlink)) {
+                        if (process.platform === 'win32' && (err.code === 'EPERM' || err.code === 'ENOENT') && _path.default.isAbsolute(symlink)) {
                             try {
                                 await _fs.promises.symlink(symlink, fileOutputPath, 'junction');
                             } catch (junctionErr) {
